src/model.py

```diff
@@ -442,18 +442,24 @@ def update_after_run(
             (state, next_execution, bot_id),
         )
 
+    @property
+    def name(self) -> str:
+        return f"{self.family}_{self.bot_id}_{self.country}"
```

Comment:
```suggestion
        return f"{self.family}_{self.tracker_id}_{self.country}"
```

Not sure if a typo or a misunderstanding but this is what I meant in the original comment about bot vs tracker hierarchy.

Bot identifier provides absolutely no context and it even makes it more confusing when mixed in with the tracker id in other places. Bot country is much better and uniquely identified the bot among others within a specific tracker

Reply:
Neither typo nor misunderstanding, just :pill: late at night and not enough testing. Thanks for spotting and sorry for pingponging it so long :thinking: